### ISSUE TITLE
FormData in POST requests crash app

### ISSUE THREAD

**Issue by Watersdr** [alexbrazier/react-native-network-logger#18]
**Describe the bug**
If there is a request that contains FormData in the body, navigating to the details section of that request crashes the app.

**Expected behavior**
FormData should be parsed correctly and displayed to the user.

**Screenshots**
![image](https://user-images.githubusercontent.com/6473620/88593238-42bcda00-d02d-11ea-9639-768fff81db0f.png)

**Platform:**
- [x] iOS
- [x] Android

**Additional context**
I believe the issue lies in this method that's inside `NetworkRequestInfo.ts`:
```js
  private stringifyFormat(data: string) {
    try {
      return JSON.stringify(JSON.parse(data), null, '\t');
    } catch (e) {
      return data;
    }
  }
```
`JSON.parse` throws an error, it's caught, then the FormData is passed onto `<LargeText>` as a child but since it's an object still, it crashes

**Comment by alexbrazier** [comment]
Hi @Watersdr thanks for opening the issue. The issue should now be fixed and released in [1.5.0](https://github.com/alexbrazier/react-native-network-logger/releases/tag/v1.5.0). Let me know if it's working for you.

At the moment I've formatted the result as JSON, but I'm not sure if it would make more sense to format it differently - would be good to hear your thoughts on it?

I've also opened this issue to track the share as curl feature for form data: #19
